compiler-rs/clients_schema/src/transform/expand_generics.rs

```diff
@@ -486,6 +486,7 @@ mod tests {
     use super::*;
 
     #[test]
+    #[ignore]
```

Comment:
So the test runs, but there's a diff regarding common query parameters that I have no idea how to fix:

```diff
--- compiler-rs/clients_schema/test-output/schema-no-generics-canonical.json	2024-10-30 09:19:45
+++ compiler-rs/clients_schema/test-output/schema-no-generics-new.json	2024-10-30 09:19:45
@@ -42873,183 +42873,6 @@
       ]
     },
     {
-      "kind": "interface",
-      "name": {
-        "namespace": "_spec_utils",
-        "name": "CommonCatQueryParameters"
-      },
-      "description": "Implements a set of common query parameters all Cat API's support.\nSince these can break the request structure these are listed explicitly as a behavior.",
-      "specLocation": "_spec_utils/behaviors.ts#L86-L132",
-      "properties": [
-        {
-          "name": "format",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "string"
-            }
-          },
-          "required": false,
-          "description": "Specifies the format to return the columnar data in, can be set to\n`text`, `json`, `cbor`, `yaml`, or `smile`.",
-          "serverDefault": "text"
-        },
-        {
-          "name": "h",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_types",
-              "name": "Names"
-            }
-          },
-          "required": false,
-          "description": "List of columns to appear in the response. Supports simple wildcards."
-        },
-        {
-          "name": "help",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "boolean"
-            }
-          },
-          "required": false,
-          "description": "When set to `true` will output available columns. This option\ncan't be combined with any other query string option.",
-          "serverDefault": false
-        },
-        {
-          "name": "local",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "boolean"
-            }
-          },
-          "required": false,
-          "description": "If `true`, the request computes the list of selected nodes from the\nlocal cluster state. If `false` the list of selected nodes are computed\nfrom the cluster state of the master node. In both cases the coordinating\nnode will send requests for further information to each selected node.",
-          "serverDefault": false
-        },
-        {
-          "name": "master_timeout",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_types",
-              "name": "Duration"
-            }
-          },
-          "required": false,
-          "description": "Period to wait for a connection to the master node.",
-          "serverDefault": "30s"
-        },
-        {
-          "name": "s",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_types",
-              "name": "Names"
-            }
-          },
-          "required": false,
-          "description": "List of columns that determine how the table should be sorted.\nSorting defaults to ascending and can be changed by setting `:asc`\nor `:desc` as a suffix to the column name."
-        },
-        {
-          "name": "v",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "boolean"
-            }
-          },
-          "required": false,
-          "description": "When set to `true` will enable verbose output.",
-          "serverDefault": false
-        }
-      ]
-    },
-    {
-      "kind": "interface",
-      "name": {
-        "namespace": "_spec_utils",
-        "name": "CommonQueryParameters"
-      },
-      "description": "Implements a set of common query parameters all API's support.\nSince these can break the request structure these are listed explicitly as a behavior.\nIts up to individual clients to define support although `error_trace` and `pretty` are\nrecommended as a minimum.",
-      "specLocation": "_spec_utils/behaviors.ts#L50-L84",
-      "properties": [
-        {
-          "name": "error_trace",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "boolean"
-            }
-          },
-          "required": false,
-          "description": "When set to `true` Elasticsearch will include the full stack trace of errors\nwhen they occur.",
-          "serverDefault": false
-        },
-        {
-          "name": "filter_path",
-          "type": {
-            "kind": "union_of",
-            "items": [
-              {
-                "kind": "instance_of",
-                "type": {
-                  "namespace": "_builtins",
-                  "name": "string"
-                }
-              },
-              {
-                "kind": "array_of",
-                "value": {
-                  "kind": "instance_of",
-                  "type": {
-                    "namespace": "_builtins",
-                    "name": "string"
-                  }
-                }
-              }
-            ]
-          },
-          "required": false,
-          "description": "Comma-separated list of filters in dot notation which reduce the response\nreturned by Elasticsearch."
-        },
-        {
-          "name": "human",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "boolean"
-            }
-          },
-          "required": false,
-          "description": "When set to `true` will return statistics in a format suitable for humans.\nFor example `\"exists_time\": \"1h\"` for humans and\n`\"eixsts_time_in_millis\": 3600000` for computers. When disabled the human\nreadable values will be omitted. This makes sense for responses being consumed\nonly by machines.",
-          "serverDefault": false
-        },
-        {
-          "name": "pretty",
-          "type": {
-            "kind": "instance_of",
-            "type": {
-              "namespace": "_builtins",
-              "name": "boolean"
-            }
-          },
-          "required": false,
-          "description": "If set to `true` the returned JSON will be \"pretty-formatted\". Only use\nthis option for debugging only.",
-          "serverDefault": false
-        }
-      ]
-    },
-    {
       "kind": "type_alias",
       "name": {
         "namespace": "_spec_utils",
\ No newline at end of file
```